compressor/tests/test_offline_url_placeholders.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from compressor.exceptions import OfflineGenerationError
+from compressor.management.commands.compress import Command as CompressCommand
+import compressor.tests.test_offline
+
+
+module_dict = globals()
+
+
+for key in dir(compressor.tests.test_offline):
+    value = getattr(compressor.tests.test_offline, key)
+    if (
+        key not in ('OfflineCompressStaticTemplateTagTestCase', 'TestCompressCommand') and
+        # Find all the TestCase subclasess except the TestCase itself
+        isinstance(value, type) and issubclass(value, TestCase) and value is not TestCase
+    ):
+        new_class = type(key, (value,), {})
+        module_dict[key] = override_settings(COMPRESS_OFFLINE_USE_URL_PLACEHOLDER=True)(new_class)
```

Comment:
There is `compress/tests/test_offline.py` file, it has ~60 tests. I want to re-use them: to run them all but with `settings.COMPRESS_OFFLINE_USE_URL_PLACEHOLDER=True`.

I'm dynamically creating subclasses with overridden setting.